src/components/WorkOrders/Filter/WorkOrdersFilter.js

```diff
@@ -10,6 +12,16 @@ const WorkOrdersFilter = ({
   appliedFilters,
   clearFilters,
 }) => {
+  const { user } = useContext(UserContext)
+
+  const statusFilterOptions = () => {
+    if (user && user.hasContractorPermissions) {
+      return filters.Status.filter((status) => status.key !== '1010')
```

Comment:
I don't think it's necessarily something for this PR, but this makes me wonder if we should make a move to capture all of these status codes so we don't have to sometimes search around for their meaning - perhaps in environment variables? Or a config object with keys like `"authorisation_pending_approval"` mapped to the code and label text. That might make the views and cypress tests more readable and make things like typos less likely.

Reply:
Ah so I did create a `status-codes.js` file which i've forgotten to use here but yeah we should probably move to something a bit better like you suggest